tests/test/test_history.py

```
@@ -0,0 +1,31 @@
+import pytest
+
+@pytest.mark.usefixtures("login_with_volunteer")
+@pytest.mark.smoke
+def test_history_page_load(driver, history_page):
```

Ruff (0.15.2)

[warning] Unused function argument: `driver`

(ARG001)